<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Memory_game</class>
 <widget class="QWidget" name="Memory_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
   <item>
    <layout class="QHBoxLayout" name="top_layout">
     <item>
      <spacer name="spacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="player_points_layout" rowstretch="1,1,2">
       <item row="0" column="1">
        <widget class="QLabel" name="player_1_points">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="player_2_points">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="player_2_lbl">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hráč 2: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="player_1_lbl">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hráč 1: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="player_turn_layout">
       <item>
        <widget class="QLabel" name="turn_lbl">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Na tahu:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player_turn_lbl">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hráč 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="field"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
